cosmos/operators/local.py

```diff
@@ -112,12 +112,14 @@ def __init__(
         self,
         profile_config: ProfileConfig,
         install_deps: bool = False,
+        emit_datasets: bool = False,
         callback: Callable[[str], None] | None = None,
         should_store_compiled_sql: bool = True,
         **kwargs: Any,
     ) -> None:
         self.profile_config = profile_config
         self.install_deps = install_deps
+        self.emit_datasets = emit_datasets
```

Comment:
OK. I can see that I did not check far enough back in the class hierarchy and emit_datasets indeed does default to True. I will remove my suggested logic as well as the docstring.

Since emit_datasets does indeed default to True the problem must lie elsewhere. Should I then open a new issue describing the problem without a proposed solution, or is it better to edit the existing issue?

Reply:
I think you can keep your original issue open since this PR doesn't fix the issue you're experiencing, although it fixes the docstring that was a source of confusion - thanks!

Reply:
Ok will do